docs/_doc/Gas-Price-Oracle.md

```diff
@@ -1,22 +0,0 @@
----
-title: Gas price oracle
```

Comment:
Same here @fjl I don't see this replaced, is it needed?

Reply:
The info on this page was out of date. Our gas price oracle was simplified and has fewer parameters now. It doesn't really need its own page.